README.md

```diff
@@ -1,13 +1,13 @@
 fxa-content-server-l10n
 =======================
 
-[![CircleCI](https://circleci.com/gh/mozilla/fxa-content-server-l10n/tree/master.svg?style=svg)](https://circleci.com/gh/mozilla/fxa-content-server-l10n/tree/master)
+[![Tests](https://github.com/mozilla/fxa-content-server-l10n/actions/workflows/test.yaml/badge.svg)](https://github.com/mozilla/fxa-content-server-l10n/actions/workflows/test.yaml)
```

Comment:
This badge won't work until the action is run once. Here's the fork's badge https://github.com/flodolo/fxa-content-server-l10n/actions/workflows/test.yaml/badge.svg